README.md

```diff
@@ -1289,6 +1289,27 @@ Other Style Guides
 
   - [15.4](#15.4) <a name='15.4'></a> For more information see [Truth Equality and JavaScript](http://javascriptweblog.wordpress.com/2011/02/07/truth-equality-and-javascript/#more-2108) by Angus Croll.
 
+  - [15.5](#15.5) <a name='15.5'></a> Ternaries should not be nested and generally be single line expressions.
```

Comment:
But if the line is long, you can just use intermediate variables instead of a ternary. eg,
```js
const nameWithNick = `${this.firstName} "${this.nickname}" ${this.surname}`;
const nameWithoutNick = `${this.firstName} ${this.surname}`;
const name = this.nickname ? nameWithNick : nameWithoutNick;
```
It's still three lines, but the ternary is on a single line, and you have three statements to step through with a debugger instead of one.